testing-modules/junit-5/src/test/java/com/baeldung/conditional/ConditionalAnnotationsTest.java

```diff
@@ -0,0 +1,119 @@
+package com.baeldung.conditional;
+
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.*;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+public class ConditionalAnnotationsTest {
+    @Test
+    @EnabledOnOs({OS.WINDOWS, OS.MAC})
+    public void shouldRunBothWindowsAndMac() {
+        //...
```

Comment:
It's OK to have the //... in the article snippets, but in the GitHub code, let's at least add a println statement.